app/Models/Membership.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Models;
+
+use App\Models\Membership\MembershipPivot;
+use App\User;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Illuminate\Support\Collection;
+
+class Membership extends Model
+{
+    const IDENT_DIVISION = 'DIV';
+    const IDENT_REGION = 'REG';
+    const IDENT_INTERNATIONAL = 'INT';
+    const IDENT_VISITING = 'VIS';
+    const IDENT_TRANSFERING = 'TFR';
+
+    public $timestamps = false;
+
+    protected $casts = [
+        'primary' => 'bool',
+        'can_have_secondaries' => 'bool',
+    ];
+
+    public function scopePrimary(Builder $query): Builder
+    {
+        return $query->where('primary', true);
+    }
+
+    public function scopeSecondary(Builder $query): Builder
```

Comment:
Can this just be the inverse of scopePrimary?

Reply:
E.g.

`return !$this->scopePrimary()` since they have the same return type?

Reply:
uh no because that would return a boolean and we are returning a query builder instance? I don't think there is a way to do that with scopes (for the actual attributes of course because we are returning a boolean anyway)